user/pages/40.governance/20.collectiveagreements/default.en.md

```diff
@@ -10,20 +10,30 @@ We meet on Mondays at or before 10:00 to distribute information, make plans and
+Visitors are strongly recommended to ask for a check-in round after spending around ~7 days in the space, otherwise the host will arrange one. The check-in round is an informal chat about the relationship between Kanthaus, its residents and the Visitor. It is optional for Volunteers/Members, but the Visitor needs to attend.
+
+### Quiet Hours
+Between 00:00 and 7:00 people are expected to be silent. This is especially relevant for spaces (near to) where people sleep: K20-2, K22-1 and K22-2. There may be exceptions to this general rule, but they need to be explicitly announced and agreed upon at least a day (24 hours) in advance.
+
+### Clear Table Policy
+Please help us keep commonly used tables as clear as possible. If you are done using a surface, please remove the things you were working with, so that the table can directly be used by the next person. This is especially relevant for the tables in K20-1 (Office, Kitchen, Snack Kitchen and Living Room).
 
 ### Smoking
-Please do not smoke at all within the buildings. If you smoke outside, please ensure that smoke does not come in through open doors or windows.
+Please do not smoke at all within the buildings. If you smoke outside, please ensure that smoke does not come in through open doors or windows. Do not throw cigarette butts on the ground! They can be disposed of in the black bin in the garden.
 
 ### Drinking
 The space is not a bar: please do not come here with drinking as a primary purpose and be aware that alcohol consumption will not be considered as an excuse or justification of behavior. That said, individuals may responsibly consume alcohol in the space. Please contact a Volunteer or Member with concerns or questions regarding this topic.
 
-### Accessibility
-The space is not very accessible right now, making it generally unsuitable for individuals requiring additional support (e.g. blind, wheelchair user, etc.) Individuals requiring additional support are not restricted from coming, however we may not have the infrastructure or resources to make the visit enjoyable. Feel free to contact a Member or Volunteer to get an update on the current building status or for any other enquiry.
-
 ### Animals
 The space is not very safe right now, making generally unsuitable for animals. Even if it was safe, animals are not necessarily welcome due to factors such as: allergic reactions, hair-shedding, defecation, damage to resources, biting and other undesirable behaviors. Potential animal residents will be considered on a case by case basis. Please contact a Member to discuss bringing an animal.
 
+### Casual Mode
+If there are not more than 5 residents at a given time, all rules recorded in this document are relaxed. The people are expected to talk to each other and are explicitly encouraged to find custom solutions for everything that seems overly regulated for a group that small.
```

Comment:
I woud leave this out. From my understanding it's always possible to temporarily not follow the collective agreements if nobody has resistance against it.

A bit more on the topic: if there's only few people here, they should make a reasonable effort to keep the place in a shape for other volunteers and members to feel welcome again. I guess that means the usual common sense things, mostly "no smoking inside" and "clean up after yourself", the latter possibly with slightly relaxed time frames (e.g. days instead of hours).

Reply:
But why _not_ record something we do anyways? It's about making it more transparent. I'm happy to change the exact phrasing but I think I wouldn't want to leave it out completely.